src/app/home/_component/mission_badge.tsx

```diff
@@ -0,0 +1,38 @@
+import { MoonIcon } from '@heroicons/react/20/solid';
+import { TicketIcon } from '@heroicons/react/20/solid';
+import { RocketLaunchIcon } from '@heroicons/react/20/solid';
+
+import Badge from '@/components/ui/badge';
+
+type Mission_type = 'DAILY' | 'WEEKLY' | 'SPECIAL';
+type Props = {
+  missionType: Mission_type;
+};
+
+export default function MissionBadge({ missionType }: Props) {
+  const missions = {
+    DAILY: {
+      icon: <MoonIcon className="h-3 w-3" />,
+      text: '일간 미션',
+      color: 'white',
+    },
+    WEEKLY: {
+      icon: <TicketIcon className="h-3 w-3" />,
+      text: '주간 미션',
+      color: 'white',
+    },
+    SPECIAL: {
+      icon: <RocketLaunchIcon className="h-3 w-3" />,
+      text: '특별 미션',
+      color: 'success',
+    },
+  } as const;
+
+  const { icon, text, color } = missions[missionType];
```

Comment:
`icon` 을 JSX Element로써 렌더링 해야 하기 때문에 컨벤션에 맞추어 `Icon`이라는 이름으로 가져옵시다.


```suggestion
  const { icon: Icon, text, color } = missions[type];
```